app/pandas.ipynb

```diff
@@ -208,18 +212,16 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 17,
-   "outputs": [],
-   "source": [
-    "# Write your code here"
-   ],
+   "source": "df[\"Average age\"] = df[\"Age\"].mean()",
```

Comment:
In Task 4.2, adding 'Average age' as a column with a constant value might not be the intended task. If the task is to calculate and display the average age, consider using `print(df['Age'].mean())` instead.